caddyhttp/httpserver/context.go

```diff
@@ -410,6 +411,15 @@ func (c Context) RandomString(minLen, maxLen int) string {
 	return string(result)
 }
 
+// AddLink adds a link in response for future server push
+func (c Context) AddLink(link string) string {
```

Comment:
I think so according to [this](https://golang.org/pkg/text/template/#hdr-Pipelines)

> The output of a command will be either one value or two values, the second of which has type error. If that second value is present and evaluates to non-nil, execution terminates and the error is returned to the caller of Execute.